src/util/util_vector.h

```diff
@@ -151,12 +151,18 @@ namespace dxvk {
   static_assert(sizeof(Vector4i) == sizeof(int)   * 4);
 
   inline Vector4 replaceNaN(Vector4 a) {
+    #ifdef DXVK_ARCH_X86
     Vector4 result;
     __m128 value = _mm_load_ps(a.data);
     __m128 mask  = _mm_cmpeq_ps(value, value);
            value = _mm_and_ps(value, mask);
     _mm_store_ps(result.data, value);
     return result;
+    #else
+    for (int i = 0; i < 4; i++)
+      a[i] = a[i] != a[i] ? 0.0f : a[i];
```

Comment:
`std::isnan`?

Reply:
Generates horrible code on MSVC on x86 and ARM, but it's identical for GCC/Clang because they're not shit so I am okay with using it here.